packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -856,7 +861,11 @@ Or read our docs at https://www.prisma.io/docs/concepts/components/prisma-client
      * @returns
      */
     $queryRaw(query: TemplateStringsArray | sqlTemplateTag.Sql, ...values: any[]) {
-      return createPrismaPromise((txId, lock, otelCtx) => {
+      return createPrismaPromise(async (txId, lock, otelCtx): Promise<unknown> => {
+        if (isSql(query) && query.sql === '') {
+          return null
+        }
+
         if ((query as TemplateStringsArray).raw || (query as sqlTemplateTag.Sql).sql) {
```

Comment:
This should also probably be changed the same way

```suggestion
        if ((query as TemplateStringsArray).raw !== undefined || (query as sqlTemplateTag.Sql).sql !== undefined) {
```